truncate/table_schema.go

```diff
@@ -137,38 +123,46 @@ func fetchTableSchemas(ctx context.Context, client *spanner.Client, targetTables
 	return tables, nil
 }
 
-func fetchIndexSchemas(ctx context.Context, client *spanner.Client) ([]*indexSchema, error) {
-	// This query fetches defined indexes.
-	iter := client.Single().Query(ctx, spanner.NewStatement(`
-		SELECT INDEX_NAME, TABLE_NAME, PARENT_TABLE_NAME FROM INFORMATION_SCHEMA.INDEXES
-		WHERE INDEX_TYPE = 'INDEX' AND TABLE_CATALOG = '' AND TABLE_SCHEMA = '';
-	`))
+// filterTableSchemas filters tables by excludeTables.
+// If an exclude table is cascade deletable, its parent table is also excluded.
+func filterTableSchemas(tables []*tableSchema, excludeTables []string) []*tableSchema {
```

Comment:
New function to filter fetched tableSchema by excludeTables.

Reply:
Thanks!